docs/web-apps/automated-testing/playwright/yaml.md

```diff
@@ -611,7 +611,7 @@ The version of Playwright that is compatible with the tests defined in this file
 ```
 
 :::tip
-You can also define a path to your `package.json`. This will make saucectl use the same cypress version that's defined in your projects `devDependencies` or `dependencies` map.
+You can also define a path to your `package.json`. This will make saucectl use the same playwright version that's defined in your projects `devDependencies` or `dependencies` map.
```

Comment:
```suggestion
You can also define a path to your `package.json`. This will make `saucectl` use the same Playwright version that's defined in your projects `devDependencies` or `dependencies` map.
```